### ISSUE TITLE
Make homepage content switcher a radio group for a11y

### ISSUE THREAD

**Issue by obulat** [WordPress/openverse-frontend#745]
## Problem

#716 added the content switcher buttons, but their accessibility needs to be improved.

## Description

We should make the switcher act as a radio group so that the screen reader can understand that only one choice is possible. This concern was raised [in a comment](https://github.com/WordPress/openverse-frontend/pull/716#pullrequestreview-865307041) on #716.

## Alternatives

Currently, only the selected button has the 'aria-pressed' attribute set to true, and this is not enough for clearly showing that it's a selection between options.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

**Comment by sarayourfriend** [comment]
`VItemGroup` should be able to handle this case already if the `type="radiogroup"` prop is set.
